doc/classes/CollisionObject2D.xml

```diff
@@ -68,6 +68,7 @@
 			<argument index="0" name="owner_id" type="int">
 			</argument>
 			<description>
+				Returns [code]true[/code] if one way collision is enabled for the shape owner of given index. Only way collision is a mode where only the edges that face up, relative to CollisionObject2D's rotation, will collide with other objects.
```

Comment:
Only way collision is a mode where only the edges that face up, relative to CollisionObject2D's rotation, will collide with other objects.
->Suggestion
`One way collision is a mode where only the edges that face up, relative to the rotation [code]CollisionObject2D[/code], will collide with other objects. Call [method shape_owner_set_one_way_collision] to activate this mode.`